### PR TITLE
http quiz

### PR DESCRIPTION
**Pull request by miloofcroton**

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+        name: 'bernice',
+        description: 'What a penguin!',
+        age: 7
+      }));
+    }
+    else {
+      res.end(JSON.stringify({name: 'bernice'}));
+    }
+  }
+  else if(req.method === 'GET' && urlParts[1] === 'penguins') {
```
Comment (mehtaphysical) [inline lib/app.js]: Its impossible to get to this spot in the code. This is the same condition as the above condition

Comment (miloofcroton) [reply lib/app.js]: The above one is penguin singular. What am I missing?

Comment (mehtaphysical) [reply lib/app.js]: ah right. my fault

Comment (miloofcroton) [reply lib/app.js]: it's all good!

**Pull request merged**